### PR TITLE
[Hotels] Use new Endpoint to fetch results (polling)

### PR DESCRIPTION
**Pull request by inyigo**
- First call is POST request
- Succeeding polling is GET Request

sorry for many changes, i ran prettier

### CODE REVIEW

**Comment by viettienn** [comment]
@XavierEr @S0HAIB could you review this?

File: build/bower.bundle.js [hunk 1]
```diff
@@ -180,88 +180,123 @@ module.exports = utils;
...
     },
     production: {
-      v2: 'https://srv.wego.com/v2',
-      v1: 'https://srv.wego.com'
+      v2: "https://srv.wego.com/v2",
+      v1: "https://srv.wego.com"
     }
   },
 
+  hotelEndoints: {
```
Comment (S0HAIB) [inline build/bower.bundle.js]: Typo maybe `hotelEndoints`

**Comment by inyigo** [comment]
see https://github.com/wego/wego-api-docs/issues/66

File: src/Api.js [hunk 2]
```diff
@@ -1,88 +1,123 @@
...
 
   getEnvironment: function() {
-    return this.env || Wego.ENV || 'staging';
+    return this.env || Wego.ENV || "staging";
   },
 
   searchTrips: function(requestBody, query) {
-    var uri = this.__host[this.getEnvironment()].v2 + '/metasearch/flights/searches';
+    var uri =
+      this.__host[this.getEnvironment()].v2 + "/metasearch/flights/searches";
```
Comment (wilson-alcantara) [inline src/Api.js]: can we change the `.v2` into bracket notation?

Comment (wilson-alcantara) [reply src/Api.js]: Or we can use `getHost()`

Comment (wilson-alcantara) [reply src/Api.js]: ohh changes not need as it has been updated on [branch 42](https://github.com/wego/wego-sdk-js/pull/42/files#diff-dd58b2d4133ed7d8479cd745a1390f92R233)

**Pull request merged**